addons/medical_statemachine/functions/fnc_enteredStateDeath.sqf

```diff
@@ -23,5 +23,5 @@ TRACE_3("enteredStateDeath",_this,_thisOrigin,_thisTransition);
 
 private _causeOfDeath = format ["%1:%2", _thisOrigin, _thisTransition];
 
-// could delay a frame here to fix the double killed EH, but we lose it being a "native" kill (scoreboard / rating)
-[_unit, _causeOfDeath] call EFUNC(medical_status,setDead);
+// delay a frame here to fix the double killed EH
+[{[_this select 0, _this select 1] call EFUNC(medical_status,setDead)}, [_unit, _causeOfDeath]] call CBA_fnc_execNextFrame;
```

Comment:
I'm not sure if this should be introduced as don't know if we ever determined possible side effects of this. 

There was also another fix for the double killed EH if I recall correctly.

Believe @PabstMirror would be the person to know best

Reply:
>Fixed: "Killed" Event Handler was sometimes firing twice - [FT-T149510](https://feedback.bistudio.com/T149510)
I'm assuming this will make it into 2.06

there also is some CBA code that can probably be removed: https://github.com/CBATeam/CBA_A3/blob/master/addons/xeh/fnc_preInit.sqf#L66

either way I think we should wait for at least 2.06 to be out